tests/spec/fs.lstat.spec.js

```diff
@@ -13,7 +13,6 @@ describe('fs.lstat', function() {
 
   it('should return an error if path does not exist', function(done) {
     const fs = util.fs();
-
```

Comment:
Why remove all these blank lines?  It's normally a bad idea to touch lines that aren't really part of this fix.  The reason it's generally avoided is that it will alter the history of these lines in git, making them look like they were changed in this PR, but really they weren't.

Reply:
I have changed the file again with the space